add all of the available mitigations to risks of censorship. (#232)
larseggert/draft-ietf-modpod-group-processes: draft-ietf-modpod-group-processes.md

The usual security considerations {{?RFC3552}} do not apply to this
document.

There is the potential abuse of the moderation process by moderators,
working group chairs, and potentially others that could lead to

> censorship of legitimate participation. This risk can be checked in

Reviewer: "is checked" maybe? No strong opinion.

seven ways:

1. The moderator team is required to first establish processes and

> procedures that are intended to be applied uniformly across the IETF.

Author: Split the difference...
```suggestion
procedures that are intended to apply uniformly across the IETF.
```

2. We explicitly state that minority viewpoints are not
in and of themselves abuse.
3. Moderation actions that restrict posting must be made transparent
to the affected person and to the moderation team, and must be
periodically reported to the IESG.

> 4. In the case of bans lasting longer than 14 days, the community is

Reviewer 2: ```suggestion
4. In the case of restrictions of participation privileges lasting longer than 14 days, the community is
```

also informed.
5. An appeals process is made available to anyone in the case of
disagreements.

> 6. If people believe that the IAB or IESG members are not performing

Reviewer 2: ```suggestion
6. If IETF participants believe that the IESG is not performing
```

Why include the IAB? They are just an escalation in the appeals path.
